.ebextensions/eb-options.config

```diff
@@ -1,8 +1,3 @@
 option_settings:
   aws:elasticbeanstalk:container:nodejs:
     ProxyServer: "nginx"
-    NodeCommand: "node --max_old_space_size=2000  ./dist/server.js"
-    NodeVersion: "12.14.1"
-
-  aws:elasticbeanstalk:command:
```

Comment:
you probably don't want to remove this.

Reply:
@zaro0508 , which lines you don't want me to remove? The timeout? Thanks.